hoomd/device.py

```diff
@@ -359,28 +359,29 @@ def __init__(self,
             self.num_cpu_threads = num_cpu_threads
 
 
-class Auto(_Device):
+def auto(communicator=None, msg_file=None, shared_msg_file=None, notice_level=2):
```

Comment:
Let's make the method a verb. `auto_select`?

Reply:
Done